src/app/globals.css

```
@@ -119,6 +119,172 @@ button {
   font: inherit;
 }
 
+.visually-hidden {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)